### PR TITLE
Add Apollo skeleton

### PR DESCRIPTION
**Pull request by StorytellerCZ**
Added Apollo skeleton based on the React skeleton and updated dependencies on all skeletons.
It is a very simple Apollo integration without anything fancy using the official `apollo` package.
Maybe in the future it might be a good idea to have more diversity in Apollo skeletons, but I think this is a much needed start of that process.

### CODE REVIEW

File: tools/static-assets/skel-apollo/imports/ui/Info.jsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { useQuery } from '@apollo/react-hooks';
```
Comment (filipenevola) [inline tools/static-assets/skel-apollo/imports/ui/Info.jsx:2]: I believe you need to change this as well

```suggestion
import { useQuery } from '@apollo/client';
```

**Comment by StorytellerCZ** [comment]
@filipenevola I have written an article on my thinking on Dev.to: https://dev.to/storytellercz/meteor-create-apollo-32e2
In short I see this as the basic HTTP GraphQL skeleton that most developers are familiar with (plus React). I think additional skeletons around Apollo should be coming in next version. I see something like `apollo-ddp`, to integrate the benefits of DDP, as the one with most interest from Meteor community. This skeleton though is targeted at new comers.

**Comment by StorytellerCZ** [comment]
I'll make another round of updating the skeletons dependencies tomorrow.

**Comment by filipenevola** [comment]
Thanks @StorytellerCZ

**Pull request merged**